ivy/functional/frontends/paddle/math.py

```diff
@@ -118,6 +118,12 @@ def atanh(x, name=None):
     return ivy.atanh(x)
 
 
+@with_supported_dtypes({"2.5.1 and below": ("int32", "int64")}, "paddle")
+@to_ivy_arrays_and_back
+def broadcast_shape(x, y):
+    return ivy.broadcast_shapes(*[x + y])
```

Comment:
I think you don't need to do this as the function signature in `ivy.broadcast_shapes` itself has `*` already, you may just pass in `...(x, y)`
Thanks!

Reply:
ok